lib/ftp/src/ftp.erl

```diff
@@ -2191,14 +2160,25 @@ connect2(Host, Port, IpFam, SockOpts, Timeout) ->
             Error
     end.
 
+accept_data_connection_tls_options(#state{ csock = {ssl,Socket}, tls_options = TO0 }) ->
+    case lists:keyfind(reuse_sessions, 1, TO0) of
+        {reuse_sessions,save} ->
+            TO1 = lists:keyreplace(reuse_sessions, 1, TO0, {reuse_sessions,false}),
+            {ok, [{session_id,SSLSessionId}]} = ssl:connection_information(Socket, [session_id]),
+            TO2 = lists:keystore(reuse_session, 1, TO1, {reuse_session,SSLSessionId})],
```

Comment:
[2020-07-07T08:37:44.991Z] ftp.erl:2168: syntax error before: ']'

Reply:
...deeply embarrassing, sorry. Pushed a (rebased) fix; commit relevant commit is 9153f7ad6b6c2e716478f457d257d3fc147e0ef3